hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestKeyInputStream.java

```diff
@@ -257,7 +257,7 @@ public void testSeek() throws Exception {
 
     // Seek operation should not result in any readChunk operation.
     Assert.assertEquals(readChunkCount, metrics
-        .getContainerOpsMetrics(ContainerProtos.Type.ReadChunk));
+        .getContainerOpCountMetrics(ContainerProtos.Type.ReadChunk));
```

Comment:
So this was comparing pending op count, and now it will compare completed op count.

But I don't think this would fix the test failure:

```
testSeek(org.apache.hadoop.ozone.client.rpc.TestKeyInputStream)  Time elapsed: 0.103 s  <<< FAILURE!
java.lang.AssertionError: expected:<0> but was:<-1>
  ...
  at org.apache.hadoop.ozone.client.rpc.TestKeyInputStream.testSeek(TestKeyInputStream.java:259)
```

Neither pending nor completed count should ever be -1.

Reply:
I agree it should not be -1. But if we look at completedCount it doesn't have a decrement method whereas pendingOpCount does have it. So completedCount will never decrement from 0 but pendingOpCount can. There is a problem with how we are decrementing the pending metric (we are decrementing when a reply comes and in onError call) and that should be tracked via a separate jira. There should be a better mechanism to decrement pendingOpCount.

Reply:
You are right, I forgot that the other metric is not decremented, so it will not have this problem.